src/components/signup/all-set.tsx

```diff
@@ -31,7 +31,7 @@ const AllSet = () => {
         </div>
         <Button
           className='w-full text-white bg-[#573DF5] border border-[#7F56D9] shadow-sm shadow-[#1018280D] font-semibold text-base'
-          onClick={() => router.push('/')}
+          onClick={() => router.push('/daniel')}
```

Comment:
This seems like a WIP change. Should this be:

```suggestion
          onClick={() => router.push('/')}
```